scripts/variantstore/wdl/extract/create_ranges_cohort_extract_data_table.py

```diff
@@ -72,13 +72,19 @@ def load_sample_names(sample_names_to_extract, fq_temp_table_dataset):
     return fq_sample_table
 
 
-def get_all_sample_ids(fq_destination_table_samples):
-    sql = f"select sample_id from `{fq_destination_table_samples}`"
+def get_all_sample_ids(fq_destination_table_samples, only_output_vet_tables, fq_sample_mapping_table):
```

Comment:
oops this was not doing the right thing before? 😬

Reply:
🙈